site/pages/theme-topic-en.hbs

```diff
@@ -0,0 +1,108 @@
+---
+{
+	"title": "[Theme - Topic title]",
+	"language": "en",
+	"altLangPrefix": "theme-topic",
+	"pageType": "theme-topic",
+	"pageclass": "page-type-nav",
+	"breadcrumb": [
+		{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+		{ "title": "[Theme Name]", "link": "theme-topic-en.html" }
+	],
+	"dateModified": "2020-08-20",
+	"share": "true",
+	"deptfeature": true,
+	"contributors": true,
+	"noMainContainer": true
+}
+---
+<div class="container">
+	<div class="row profile">
+		<div class="col-md-6">
+			<h1 property="name" id="wb-cont">{{{title}}}</h1>
+			<p>1-2 sentences that describe the topics and top tasks that can be accessed on this page.</p>
+		</div>
+		<div class="col-md-6 mrgn-tp-sm hidden-sm hidden-xs">
+			<img src="{{assets}}/img/825x200.jpg" alt=""/>
```

Comment:
add class "img-responsive" https://wet-boew.github.io/wet-boew-styleguide/index-en.html

Reply:
Added to both fr/en